README.adoc

```diff
@@ -321,7 +321,10 @@ Panic! Then completely regenerate the site via:
 
 == License
 
-The content of this repository is released under the link:http://www.apache.org/licenses/LICENSE-2.0.txt[ASL 2.0].
+The articles, blog posts and other content of this repository is released under the link:http://creativecommons.org/licenses/by-sa/3.0/[Creative Commons Attribution Share-Alike 3.0 Unported (CC BY-SA 3.0)].
```

Comment:
Either drop the 'the' and just "released under Creative Commons Attribution Share-Alike 3.0 Unported (CC BY-SA 3.0)" or keep the 'the' but then it is most likely better with "released under the Creative Commons Attribution Share-Alike 3.0 Unported (CC BY-SA 3.0) license."